### PR TITLE
Face Recognition and Facial Expression Recognition in Python

### LINKED ISSUE #858: Face Recognition and Facial Expression Recognition in python
**This template is strictly for requesting New Scripts, if you want a new feature (automating this Repo,**
**a change in structure) in the Repo,** 
**or you want to report a Bug. There are other templates which you should follow**

# Description
I would like to create a script of Face recognition which can detect the face of a person and show his/her name from the database and another script that will recognize the facial expression of a person at real time.

## Language

- [ ] Bash
- [ ] Go
- [ ] Javascript
- [ ] Powershell
- [x] Python
- [ ] Other (Please specify)

## Checklist:

- **Details on the Script**

I will be working as a GSSOC'21 participant, so I will need a label of gssoc. It would be great if I get assigned on this issue @HarshCasper 

| name            | about                                                  | labels  | assignees                                                      |
| --------------- | ------------------------------------------------------ | ------- | -------------------------------------------------------------- |
| Face Recognition and Facial Expression Recognition | I would like to create a script of Face recognition which can detect the face of a person and show his/her name from the database and another script that will recognize the facial expression of a person at real time. | gssoc | @Debashish-hub |

For inspiration, see [this](https://github.com/anubhavshrimal/Face-Recognition).

### PR DESCRIPTION
**Pull request by Debashish-hub**
# Description

Added Face Recognition and Facial Expression Recognition using Python

Fixes #858

## Type of change

_Please delete options that are not relevant._

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [x] Documentation Update

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

Hello! It would be great if my work will get merged with gssoc'21 label @HarshCasper @sohamsshah @seema1711 @vybhav72954
Let me know if any changes I need to make.

Thank you

### CODE REVIEW

File: Python/Face-Recognition/readme.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+
+
+Now you can see the faces with name on the screen.
+
+![Face-Recognition](https://github.com/Debashish-hub/Rotten-Scripts/blob/master/Python/Face-Recognition/ezgif.com-gif-maker.gif)
+
+## Author
+
+[Debashish kumar sahoo](https://github.com/Debashish-hub)
```
Comment (vybhav72954) [inline Python/Face-Recognition/readme.md:54]: Is this line correct?

**Review by vybhav72954 — CHANGES_REQUESTED** [review]

File: Python/Face-Recognition/readme.md (hunk 1 above)
Comment (Debashish-hub) [reply Python/Face-Recognition/readme.md:54]: Are you talking about the author?

Comment (vybhav72954) [reply Python/Face-Recognition/readme.md:54]: Yes

Comment (Debashish-hub) [reply Python/Face-Recognition/readme.md:54]: Yes, that's me.

Comment (vybhav72954) [reply Python/Face-Recognition/readme.md:54]: If that's the case can you tell me, who [this](https://github.com/anubhavshrimal/Face-Recognition) repo has the same to same scripts, including the comments and docstrings and what not?

Comment (Debashish-hub) [reply Python/Face-Recognition/readme.md:54]: I have got the codes from somewhere and applied it on my system to check its working.

Comment (vybhav72954) [reply Python/Face-Recognition/readme.md:54]: The code is Plagiarised from [here](https://github.com/anubhavshrimal/Face-Recognition).

**For that very reason am closing the PR.**

@Debashish-hub Either, 

1. in the README mention the exact name of the Author
2. Make some changes in the existing code to make it better
3. Add some personal touches to the script

Comment (vybhav72954) [reply Python/Face-Recognition/readme.md:54]: I don't mind using old code at all, but either you should explicitly mention the Source, or do something different.

**Pull request closed without merging**